packages/fuzzer/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
-cmake_minimum_required(VERSION 3.10) # shipped with Ubuntu 18.04
+# Suggested settings by cmake-js
+cmake_minimum_required(VERSION 3.15)
+cmake_policy(SET CMP0091 NEW)
+cmake_policy(SET CMP0042 NEW)
```

Comment:
Can you add comments on why these additions are necessary?

Reply:
Added the explanation from the cmake-js release log. Is  that enough?

Reply:
These changes affect macOS and Windows respectively. Can we move them down the file where we perform os-dependant settings?

Reply:
done